labs/firefly.md

```diff
@@ -0,0 +1,30 @@
+---
+layout: default
+title: FireFly
+parent: Labs
+---
+# Lab Name
+FireFly
+
+# Short Description
+FireFly is a multiparty system for enterprise data flows, powered by blockchain.
+
+# Scope of Lab
+FireFly is designed to make building blockchain apps quick and easy, by allowing developers to focus on business logic, instead of infrastructure and cryptography. Enterprise use cases often require a mix of off-chain and on-chain activities to implement a use case end-to-end. FireFly provides a set of pre-integrated runtimes and an API to build event-driven, multi-party applications with a combination of off-chain data flows and on-chain transactions.
+
+FireFly is designed to be completely modular, allowing components at each layer to be swapped out for a different implementation, including the messaging system, the database, and even the blockchain itself.
+
+
+# Initial Committers
+- https://github.com/peterbroadhurst
+- https://github.com/nguyer
+- https://github.com/gabriel-indik
+- https://github.com/shorsher
+- https://github.com/awrichar
+
+# Pre-existing repositories
+- https://github.com/kaleido-io/firefly
+- https://github.com/kaleido-io/firefly-cli
+- https://github.com/kaleido-io/firefly-ui
+- https://github.com/kaleido-io/ethconnect
```

Comment:
Will we be renaming this repository to something like firefly-ethconnect?

Also, will we have resolved licensing issues with this particular repo prior to transferring into labs?

Reply:
Yes, we can rename this repo as a part of moving it over.

The licensing issues have been resolved in `ethconnect` now.